script.js

```diff
@@ -1,12 +1,239 @@
 //You can edit ALL of the code here
+
+//create certain elements globally to not need to keep passing elements
+let results = 0;
+let allShows;
+
+const rootElem = document.getElementById("root");
+let headingEl = createDiv("headingEl", "tv show project"); //level 100
+rootElem.appendChild(headingEl);
+let searchBarEl = createDiv("searchBar"); //level 200 search bar creation
+rootElem.appendChild(searchBarEl);
+let inputContainerEl = createDiv("inputContainer");
+searchBarEl.appendChild(inputContainerEl);
+let searchInputEl = createInput("searchInput", " Search for Episode");
+inputContainerEl.appendChild(searchInputEl);
+let searchDescEl = createSearchDiv("searchDescEl");
+inputContainerEl.appendChild(searchDescEl);
+//level 300 400
+let showsSelectEl = createSelect(`Click for All Shows`);
+searchBarEl.appendChild(showsSelectEl);
+showsSelectEl.setAttribute("id", "showNameSelect");
+let searchSelectEl = createSelect(`Click for All Episodes`);
+searchBarEl.appendChild(searchSelectEl);
+//level 100 display of episode cards creation
+const showsEl = document.createElement("section");
+showsEl.className = "showsEl";
+rootElem.appendChild(showsEl);
+
+showsSelectEl.addEventListener("change", function () {
+  if (this.value === "Click for All Shows") {
+    showEpisodes(episodeList);
+  } else {
+    let SHOW_ID = allShows[this.value].id;
+    let apiUrl = `https://api.tvmaze.com/shows/${SHOW_ID}/episodes`;
+    //https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionsCollection
+    searchSelectEl.options.length = 1;
+    getDataFromAPI(apiUrl);
+  }
+});
```

Comment:
It might be useful putting this as part of setup function or making an intialization function to wrap all these things in.
I also like that you've used functions to take create dom element - well done!